The app was crashing after I solved the merge conflicts. Fixed it. Also cleaned up the navigation drawer code a bit.

Adithya2907 opened the pull request:
### Description
Fixed crashing of app due to merging to two PRs.

Please ensure the following and put a check accordingly

- [ ] Ran checks with `./gradlew ktlintCheck` command

- [ ] Squashed multiple commits (_if irrelevant_) into one commit

Commit:
The app was crashing after I solved the merge conflicts. Fixed it. Also cleaned up the navigation drawer code a bit.

devansh-299 commented on app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt:
```diff
@@ -12,40 +12,36 @@ import club.devsoc.scanf.databinding.ActivityMainBinding
 import com.google.android.material.navigation.NavigationView
 
-class MainActivity : AppCompatActivity(), View.OnClickListener {
+class MainActivity : AppCompatActivity(), View.OnClickListener, NavigationView.OnNavigationItemSelectedListener {
```
@Adithya2907 currently it's not enforced in the build checks to limit max characters to 100 but it is advised not to go beyond this. Please update the PR accordingly and resolve the merge conflicts

Adithya2907 replied:
@devansh-299 I've made the change, and also resolved the merge conflicts. Is there any specific reason as to why code must not exceed 100 characters in a line?

devansh-299 replied:
@Adithya2907 There's no specific reason other than it makes code more readable. Different limits are used in different organizations like many use a limit of 80. I have seen 100 being adopted by many other organizations that's why I wanted to enforce this limit.
Also please squash your commits into one commit.

Adithya2907 replied:
Oh alright. And to squash commits into one, do I need to send a new PR?

devansh-299 replied:
> Oh alright. And to squash commits into one, do I need to send a new PR?

Nope, try to do this within this PR only

Adithya2907 replied:
@devansh-299 I'm not sure how I should squash commits into one. Do you have any resources which I can refer to?

Adithya2907 committed:
Merge branch 'master' into master

Commit:
Home Page done, and no conflicts in this file.

 Conflicts:
	app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt

Commit:
Merge remote-tracking branch 'origin/master'

# Conflicts:
#	app/src/main/java/club/devsoc/scanf/view/activity/MainActivity.kt
